concepts/time/introduction.md

```diff
@@ -4,4 +4,14 @@ In Go, functionality for working with times is provided by the `time` package. T
 
 To work with time, you will usually call a method on a `Time` instance, but there are also some functions called on the `time` package itself.
 
-Parsing strings works a little differently in Go. Many languages use a format string with various codes like YYYY for four-digit year, MM for two-digit month, etc. Go instead uses an exact date, `Mon Jan 2 15:04:05 -0700 MST 2006`, to parse strings. You can rearrange these components or omit them as necessary, and you can spell out month and day names, or use their number values, but in order for Go to understand the parts of the string you are parsing, the corresponding parts of the format string must be from this exact date.
+Date-time strings are represented a bit differently in Go. In many languages and libraries, developers represent their desired date-time string format with the [ISO-8601 standard](https://en.wikipedia.org/wiki/ISO_8601). In this syntax, developers must express their input/output date-time string like `YYYY mm:ss`. If developers want the library to represent `5:43 PM` as `17:43` they pass the format string `"HH:mm"`. The `time` package in Go, allows developers to instead represent a specific time in their desired format.
```

Comment:
I don't think ISO 8601 says anything about the syntax (letters) you should use when converting a date to a string. In my understanding, the standard only defines that if you parse a date like "2022-12-01" give that it is supposed to follow the standard then 2022 is the year, 12 the month etc. The letters are only used to describe that in a generic way but are not themselves standardized.
Happy to be provided with some source that says otherwise.

Reply:
I'm pretty sure that ISO 8601 defines that these (mm/MM) be used though not sure if the standard coined these: https://www.loc.gov/standards/datetime/iso-tc154-wg5_n0038_iso_wd_8601-1_2016-02-16.pdf

Reply:
Thanks a lot for link! I had trouble finding it myself. Still I find the text not super clear tbh.

In 3.3 there is this part first which makes it sound like the letters are just for writing down the standard.

> To define the date and time representations permitted by this International Standard, use is made of
date and time format representations in which specific characters are used to represent digits or other
characters in date and time representations.

And then there is this bit
> By mutual agreement the parties in information interchange may transfer the date and time format
representations. Only the date and time format representations permitted by this International
Standard shall be used.

which makes it sound more like what you are saying that the standard can also be used for standardizing the representation string itself.